focus_converter_base/focus_converter/conversion_configs/aws-cur/billing_period_end_S001.yaml

```
@@ -1,4 +1,4 @@
 plan_name: convert  bill/BillingPeriodStartDate to BillingPeriodEnd
```

Just for plan name concision 
```suggestion
plan_name: convert  bill/BillingPeriodEndDate to BillingPeriodEnd
```

---

Oh thank you so much, this is super helpful.